conf/defaults.ini

```diff
@@ -460,6 +460,7 @@ allow_sign_up = true
 client_id = some_id
 client_secret =
 scopes = user:email
+no_scope_parameter = false
```

Comment:
With my other comment in mind perhaps the naming of this can be improved? Perhaps `empty_scopes`?